### DISCUSSION TITLE
Github Actions workflow  stuck in Starting Jobs, Not moving ahead

### DISCUSSION THREAD

**Discussion by AdityaMehtaIselect** [community/community#125010] [Actions] [answered]
### Select Topic Area

Question

### Body

GitHub Actions workflow  stuck in *Starting Jobs*, Not moving ahead

**Comment by bdach** [comment]
+1.

https://github.com/ppy/osu-queue-score-statistics/actions/runs/9174420872/job/25225248217?pr=267

^ This run has all of the output I'd expect from a successful run (even concludes with "Post job cleanup"), but is formatted all weird (where are the usual collapsible headings?) and also is stuck in not-started status.

https://github.com/ppy/osu/actions

^ Several runs here are also stuck in not-started state despite their constituent parts concluding.

https://github.com/ppy/osu/actions/runs/9174423125

Every single job in this workflow was canceled, but the whole workflow is showing as "in progress".

**Comment by OlegRa** [comment]
Workflows are stuck in the `Queued` state for private and public repos.

**Comment by muawiakh** [comment, marked as answer]
The steps are being executed but the UI doesn't catch up. Might be related to: https://www.githubstatus.com/incidents/2bglpcjvg7xb

**Reply by PrzemekZglinicki** [reply]
Probably it is related, but unfortunately the problem isn't resolved yet

**Comment by luigibertaco** [comment]
Stuck at `Job is about to start running on the hosted runner: GitHub Actions * (hosted)`

**Reply by CHIRAGAGARWAL123** [reply]
Same for me

**Reply by AndersNestech** [reply]
We have jobs apparently stuck at `Job is about to start running on the runner: *** (organization)` as well as on `Job is waiting for a runner from *** to come online.`
"Apparently" as there does appear to be some progress, though at a glacial rate (>10 min)

**Comment by LouAdrienFabreSMW** [comment]
All jobs are stuck a "Queued" state for us, even though they are actually processing, its just the completion status of the jobs that is not being updated

**Comment by ChrisEllisSains** [comment]
We're seeing the same issue, github has opened an incident now: https://www.githubstatus.com/incidents/nj47vccwm2zj

**Comment by gipo355** [comment]
Same issue, all jobs stuck as queued, can't delete workflows aswell

**Comment by nadia-elemica** [comment]
Same issue for me

**Comment by grandmas-favorite** [comment]
Hi everyone! 

As @muawiakh cited, these issues are likely related to [this incident](https://www.githubstatus.com/incidents/nj47vccwm2zj) that we are currently investigating. I will update the thread when there are any updates! 

Current Status: Investigating reports of degraded performance

**Reply by grandmas-favorite** [reply]
The incident has been resolved and Actions is operating normally:
- [Incident with Actions](https://www.githubstatus.com/incidents/nj47vccwm2zj)

**Comment by Navedboss** [comment]
Trnnd bdbdhsk dnd s jbsvdjj. Dhhsbnksnsv sbsbjd. Hd dj dj djn fol fkoooxv jdn k ekktd j snake nnbsyun snap SNB bdh dhhsbnksnsv SBB sbsbjd jdn dbdnll snn snn send bdh nd ndn snsmxo ndn bandh sbsbjd sbsbjd sand sbsbjd dbdb bdbdhsk sbdbj me dbdnll dnxnxnxj dn hacks kfnjd dj bdbdhsk dntbjfy hacks hacks hacker of my derrr

**Comment by CHIRAGAGARWAL123** [comment]
The issue is happening again can someone look into it again?

**Comment by TyranVdMerwePaytec** [comment]
The issue is happening again.

**Comment by Jortiz72** [comment]
Requesting iOS Node App TestFlight access.

**TestFlight email:** insomniac357@gmail.com

**Use case:** Location + camera node for home automation.
